deployer/aws-lambda/content-origin-response/index.js

```diff
@@ -40,5 +43,9 @@ exports.handler = async (event) => {
     ];
   }
 
+  response.headers["content-security-policy-report-only"] = [
```

Comment:
Will this send that massive blob of headers on EVERY possible file coming from S3 thru the CDN? E.g.  https://developer.mozilla.org/static/media/triangle.e8097649.svg

Should we look at the `event` object and see what the `content-type` is or something?
Or we can look for things like `^/static` perhaps.

Reply:
True, I did not think about other assets being served through that endpoint. I will add a check!